templates/zerver/for/open-source.md

```diff
@@ -1,104 +1,92 @@
-
-* **Easy custom linkification:** Want to refer to issues or code
-  reviews with short notation like `#1234`, `T1234` — or any other
-  style of your own?  Set up any regex as a [custom linkification
-  filter](/help/add-a-custom-linkification-filter) for your
-  organization.
-
-* **Plays well with others:** With [60+ integrations](/integrations)
-  including GitHub, Travis CI, JIRA, and Stripe, chances are Zulip
-  already integrates with the tools you use every day; and our API
-  makes it easy to add more.
-
-* **Everything in its place:** Troubleshooting in chat for users or
-  new developers becomes much more manageable with Zulip's topic
-  threading: each issue in a channel can be discussed in its own
-  thread.
-
-## Great for productivity
-
-* **Clearer conversations:** With Zulip's unique model of threading by
-  topic, your community can have more than one conversation in a
-  channel at the same time, without the confusion and talking-over
-  caused by traditional models like IRC and Slack.
-
-* **Full service:** No need to settle for a limited "try it out" tier.
-  You're welcome to [host Zulip
-  yourself](https://zulip.readthedocs.io/en/latest/prod-install.html),
-  or [sign up at zulipchat.com](https://zulipchat.com/beta/) for
-  commercially-supported hosting — free for open source projects.
-
-* **Fast full-text search:** Feeling dejà vu about an issue someone's
-  reporting? Searching your full message history is a snap.
-
-* **Proven model:** [Stop by
-  chat.zulip.org](https://zulip.readthedocs.io/en/latest/chat-zulip-org.html),
-  Zulip's own development community, to see it in action for a large
-  project with ~5000 messages/week, essentially all of it productive
-  developer discussion.
-
-## Great for open communities
-
-* **Web, mobile, desktop:** Sleek, modern [apps for every
-  platform](/apps/) make Zulip an inclusive choice for everyone in
-  your community.
-
-* **Walls optional:** Take down the barrier to participation created
-  by making people interested in your community ask for an invitation
-  before they can enter — use [one
-  checkbox](/help/allow-anyone-to-join-without-an-invitation) to open
-  your Zulip organization for anyone to join.
-
-* **Badges:** Want to link to your Zulip organization with a nice
-  badge in your README?  [Done.](/help/join-zulip-chat-badge)
-
-<!--- This needs better documentation. -->
-* **GitHub auth?** [We've got that.][github-auth]
-
+The Zulip core developers have decades of combined experience building,
+running, and maintaining free and open source software. We use Zulip to keep
+contributors engaged, efficiently make decisions, and fashion the day-to-day
+experience of being a part of our project. There is no other chat product
+that comes even close to Zulip in facilitating contributor engagement,
+making efficient use of maintainers' time, and upholding the values of the
+FOSS community.
+
+If you haven't read [why Zulip](/why-zulip), read that first. We've also
```

Comment:
This needs more highlighting if we want people to actually go read it.

That shouldn't block landing this PR, because the existing page is kind of a laundry list and this isn't a regression. But it'll be a valuable followup.